operator/src/main/resources/Operator.properties

```diff
@@ -205,7 +205,8 @@ WLSDO-0012=Managed Server ''{0}'' specified in the domain resource does not exis
 WLSDO-0013=Istio is enabled and the domain resource specified to expose channel {0} in the adminServices section. The channel name default, \
 default-admin, default-secure are internal to the WebLogic Kubernetes Operator. Please create the network channel with a different name in the WebLogic Domain before \
 exposing it.
-WLSDO-0014="The introspect job had failed, retrying introspection job. Current failure count: {0}."
+WLSDO-0014="The domain {0} introspect job failed, it will automatically retry in {1} seconds."
```

Comment:
Can you add ", current number of retries = {2}" or ", total number of attempts = {2}" (where the latter == retries + 1)?

Reply:
done